lib/bpk-component-flare/index.js

```diff
@@ -18,7 +18,8 @@
 
 /* @flow */
 
-import BpkFlare from './src/BpkFlare';
+import BpkFlare, { FLARE_POINTER_DIRECTIONS } from './src/BpkFlare';
 
 export type { Props as BpkFlareProps } from './src/BpkFlare';
 export default BpkFlare;
+export { FLARE_POINTER_DIRECTIONS };
```

Comment:
I wonder if `pointerDirection` could be confusing in the future if we add support for other variants. For example 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/30267516/79201835-90238b80-7e30-11ea-8bc7-7afce7b8c45c.png">

Maybe `pointerPosition` or something is better?

Reply:
Gonna CC @tiagoengel here as we came up with the name together

Reply:
Pointer `direction` is what we use on Android for vertical direction. I think `position` is more generic and it is also what we use on Android for horizontal position, `middle|start|end`. 

I also think that `direction` is more accurate in this case as we are talking about the direction the pointer is "pointing", not really the position.

If we want to be really explicit about it we should call `pointerVerticalDirection` or something like that?

Reply:
To add `position` and `direction` can be used together on Android, e.g. pointing up and at the start, or pointing down and at the middle, etc

Reply:
I would just go with `direction` tbh @georgegillams. If we add more stuff later we'll figure it out.

Reply:
Sounds good to me 👍